### PR TITLE
AST-3003 - Build: Release PR

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Final release PR. Please check all the checklist below before merging.
---

### Checklist:
- [x] Finalize changelog.txt
- [ ] Add doc links in the changelog.txt
- [x] Change minimum addon update version - ASTRA_EXT_MIN_VER
- [x] Run to update packages        : `npm audit fix`
- [x] Run to produce minified files : `grunt minify`
- [x] Run to update Google Fonts    : `grunt google-fonts`
- [x] PHPCS - vendor/bin/phpcs
- [x] Run to update version number  : `grunt version-bump --ver=<version-number>`
- [x] Check the Background Process function version
- [x] Update - Tested Upto - style.css & readme.txt
- [x] Update - `theme-auto-version` : In `inc/compatibility/starter-content/astra-settings-export.json`
- [x] Update Readme.txt file and then run: `grunt readme`
- [x] Generate POT file             : `wp i18n make-pot wp-content/themes/astra wp-content/themes/astra/languages/astra.pot`
- [ ] Add draft Release notes on github
- [ ] Add draft changelogs to wpastra.com
- [ ] Open PR to backport branch to `next-release`
---

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-v4.1.0 (Unreleased)
+v4.1.0
 - New: Accessibility options to change color and style of highlighted element.
 - New: WooCommerce - Shop - Introducing new redirect actions on Add to cart click.
```
Comment (neerajmasai) [inline changelog.txt:3]: WooCommerce - Shop - Introducing new redirect actions on add to cart click.

File: changelog.txt [hunk 2]
```diff
@@ -19,7 +19,7 @@ v4.1.0 (Unreleased)
 - Fix: Spectra Compatibility - Spectra spacing options for applying custom margin and padding not working.
 - Fix: Content background color does not apply to Author Box in customizer preview.
```
Comment (neerajmasai) [inline changelog.txt:20]: Content background color does not apply to author box in customizer preview.

File: changelog.txt [hunk 3]
```diff
@@ -19,7 +19,7 @@ v4.1.0 (Unreleased)
...
 - Fix: Cost Calculator Builder - compatibility range slider.
-- Fix: Disable the title from elementor setting is not working as expected.
+- Fix: Disable the title from Elementor setting is not working as expected.
 - Fix: Footer Menu Alignment issue for responsive devices.
```
Comment (neerajmasai) [inline changelog.txt:23]: Footer menu alignment issue for responsive devices.

**Review by yssambare12 — DISMISSED** [review]

File: changelog.txt (hunk 3 above)
Comment (imnavanath) [reply changelog.txt]: Done

File: changelog.txt (hunk 2 above)
Comment (imnavanath) [reply changelog.txt]: Done

File: changelog.txt (hunk 1 above)
Comment (imnavanath) [reply changelog.txt]: Done

**Review by neerajmasai — DISMISSED** [review]

**Review by yssambare12 — APPROVED** [review]

**Comment by nathanrodrigues2111** [comment]
Code Analysis && Compressed Size failing

**Pull request merged**